scripts/src/main/resources/scripts/command/az

```diff
@@ -1,106 +1,77 @@
-      if [ "${?}" = 0 ]
-      then
-        doPackageInstall "${AZ_HOME}"
-      fi
-    fi
+function setConfig() {
+  az_config_dir="${DEVON_IDE_HOME}/conf/.azure"
+  az_config_export="export AZURE_CONFIG_DIR=${az_config_dir}"
+  if ! grep -q "${az_config_export}" "${DEVON_IDE_HOME}/conf/devon.properties"
+    then
+      doRunCommand "${az_config_export}"
+      echo -e "\n${az_config_export}" >> "${DEVON_IDE_HOME}/conf/devon.properties"
+      doEcho "Location of Azure's configuration file is set to ${az_config_dir}"
   fi
 }
 
-function doPackageInstall() {
-  local path_to_package="${1}"
+function setLocation() {
   if doIsWindows
   then 
-    local winPath
-    winPath=$(cygpath -w "${path_to_package}")
-    powershell.exe -Command "Start-Process msiexec.exe -verb runas -Wait -ArgumentList '/I ${winPath}\az-${AZ_VERSION}-windows.msi /QB-!'" &> /dev/null
-    if [ "${?}" = 0 ]
-    then
-      setConfigDir
-      doSuccess "Azure CLI ist installed now. Great! \nTo get started you may need to restart your current shell."
-    else
-      doError "The setup was canceled or failed. Rerun the setup to install Azure CLI."
-    fi
+    AZ_FILE="${DEVON_IDE_HOME}/software/python/Scripts/az"
   else
-    doFail "Sorry, Azure CLI installation support is not yet implemented for your OS. Please install manually or help devonfw-ide to support it for your OS by contributing a pull-request."
-  fi
-  doRunCommand "rm -rf ${path_to_package}"
-}
-    
-# Set directory for azure cli configuration file
-function setConfigDir() {
-  azureconf_dir="${DEVON_IDE_HOME}/conf/.azure"
-  azurecli_export="export AZURE_CONFIG_DIR=${azureconf_dir}"
-  if ! grep -q "${azurecli_export}" "${DEVON_IDE_HOME}/conf/devon.properties"
-    then
-      doRunCommand "${azurecli_export}"
-      echo -e "\n${azurecli_export}" >> "${DEVON_IDE_HOME}/conf/devon.properties"
-      doEcho "Location of Azure's configuration file is set to ${azureconf_dir}"
+    AZ_FILE="${DEVON_IDE_HOME}/software/python/bin/az"
   fi
 }
 
 function doRun() {
   doSetup silent
-  if command -v az &> /dev/null
+  doEcho "Running: az ${*}" 
+  if doIsQuiet
   then
-    doEcho "Running: Azure CLI ${*}"
-    setConfigDir
-    az "${@}"
+    # shellcheck disable=SC2068
+    az $@ > /dev/null
+  else
+    # shellcheck disable=SC2068
+    az $@
   fi
 }
 
-AZ_HOME="${DEVON_IDE_HOME}/updates/install/az"
-
 # CLI
+PYTHON_HOME="${DEVON_IDE_HOME}/software/python"
 case ${1} in
 "help" | "-h")
-  echo "Install Azure CLI."
+  echo "Setup or run Azure CLI."
   echo
   echo "Arguments:"
-  echo " setup                    install Azure CLI on your machine (global installation)."
-  echo " <<args>>                 call Azure CLI with the specified arguments. Call az --help for details or use Azure CLI directly as preferred."
+  echo " setup                    setup Azure CLI (install and verify)"
+  echo 
   echo
 ;;
 
 "setup" | "s")
-  doSetup "${2}"
+  doSetup setup
```

Comment:
Please revert this change. It was perfectly correct before. See also other commandlets like here:
https://github.com/devonfw/ide/blob/71c62a97bc93bff4b302dc93d9d6e01ecd210943/scripts/src/main/resources/scripts/command/mvn#L369

Otherwise it is not possible for other commandlets to call setup with parameter `silent` here.